REVIEWING.md

```diff
@@ -0,0 +1,32 @@
+# Reviewing contributed session
+
+## Activities / Challenges
+
+You will make sure that activities / challenges are of good quality:
+
+- Difficulty is properly set.
+- The topic of the activity / challenge is appropriate.
+- The solution provided is working.
+- Vulnerability and explanations are easy to understand.
+
+## Assets
+
+For assets, you need to check if all of them are used and mentioned in the content and make sure that the name is appropriate.
```

Comment:
```suggestion
Check if all of them are used and mentioned in the content.
If the assets are not original, make sure their license is compatible to [ours](./COPYING.md) and that their source is properly cited.
Verify that their names are appropriate.
```

Let's settle on a naming convention. What about `image-name.png`, i.e. using a dash between words, as opposed to an underscore, for example.

Reply:
I didn't commit this suggestion since I added also some rules.
Can you please check them and mark this as solved if you think so?